packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -267,7 +267,7 @@ class BuildInfo {
   /// Fields that are `null` are excluded from this configuration.
   Map<String, String> toEnvironmentConfig() {
     final Map<String, String> map = <String, String>{};
-    dartDefineConfigJsonMap?.forEach((String key, Object value) {
+    dartDefineConfigJsonMap.forEach((String key, Object? value) {
```

Comment:
Do we need to perform any null checks here for value? Or will it just get added as `'null'`?

Seems like that is the intention based on the linked issue but just wanted to double check